main.nf

```diff
@@ -173,9 +175,29 @@ if( params.gtf ){
         .ifEmpty { exit 1, "GTF annotation file not found: ${params.gtf}" }
         .into { gtf_makeSTARindex; gtf_makeHisatSplicesites; gtf_makeHISATindex; gtf_makeBED12;
               gtf_star; gtf_dupradar; gtf_featureCounts; gtf_stringtieFPKM }
+} else if( params.gff ){
+  gffFile = Channel
+                .fromPath(params.gff)
+                .ifEmpty { exit 1, "GFF annotation file not found: ${params.gff}" }
+  process convertGFFtoGTF {
```

Comment:
Please move the process down to later in the script, at the start of all of the processes (maybe before `process makeBED12`).

I don't _think_ it needs to be inside the conditional block - if this wasn't executed, then the `gffFile` channel won't exist, and the process won't run. But it's worth checking this.